src/main/java/byteceps/ui/UserInterface.java

```diff
@@ -22,7 +22,8 @@ public UserInterface(InputStream in, PrintStream out) {
     }
 
     public static void printMessage(String message) {
-        System.out.printf("[ByteCep]> %s", message);
+        System.out.printf("[ByteCeps]> %s" + System.lineSeparator(), message);
```

Comment:
Could utilize printf() by doing something like this
```suggestion
        System.out.printf("[ByteCeps]> %s%s", message, System.lineSeparator());
```